dub.sdl

```diff
@@ -60,6 +60,7 @@ subPackage {
   versions "GC"
   versions "NoMain"
   sourcePaths "src/dmd"
+  buildRequirements "disallowDeprecations"
```

Comment:
Please don't hardcode this into the package, it incurs immediate breakage on deprecations for people that cannot update their compiler.